routes/drones.js

```diff
@@ -1,36 +1,50 @@
 const express = require('express');
 const router = express.Router();
 
+const droneController = require("./../controllers/droneController")
+
 // require the Drone model here
 
 router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+    // Iteration #2: List the drones
+    // ... your code here
 });
 
+router.get("/drones", droneController.getAllDrones)
+    //router.post("/drones", droneController.getAllDrones)
+
+
 router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+    // Iteration #3: Add a new drone
+    // ... your code here
 });
 
 router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+    // Iteration #3: Add a new drone
+    // ... your code here
 });
 
+router.get("/drones/create", droneController.viewCreateDrone)
+router.post("/drones/create", droneController.viewCreateDrone)
```

Comment:
aca ya lo modifique en el comentario anterior tu controller es otro nos da el error por que esta duplicado 👀 
```suggestion
router.post("/drones/create", droneController. createDrone)
```